policies/common/Cluster.yaml

```diff
@@ -14,11 +15,16 @@ spec:
       apiCall:
         urlPath: "/apis/release.giantswarm.io/v1alpha1/releases"
         jmesPath: "items[?metadata.name == 'v{{ request.object.metadata.labels.\"release.giantswarm.io/version\" }}'] | [0]"
+    preconditions:
+      any:
+      - key: "{{ request.object.metadata.labels.\"release.giantswarm.io/version\" }}"
+        operator: Equals
+        value: "20.0.0"
```

Comment:
So we only apply this for release version `20.0.0` - all other releases are ignored.